script.js

```diff
@@ -2,11 +2,164 @@
 function setup() {
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
+  displayResults();
+  selectEpisode();
 }
 
-function makePageForEpisodes(episodeList) {
+function makePageForEpisodes(episodesList) {
   const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+
+  let ourSection = document.createElement("section");
+  ourSection.classList.add("allEpisodes")
+  rootElem.appendChild(ourSection);
+
+  for (let episodeIndex = 0; episodeIndex < episodesList.length; episodeIndex++) {
+    let episodeContent = document.createElement("article");
+    episodeContent.classList.add("episodeWrapper");                                               //Can I make the 'class' name same as the variable name since we can not use the variable name in CSS
+
+    let episodeTitle = document.createElement("h3");
+    episodeTitle.classList.add("episodeName");
+
+    let episodeImage = document.createElement("img");
+    episodeImage.classList.add("episodePicture")
+
+    let episodeSummary = document.createElement("div");
+    episodeSummary.classList.add("episodeDescription")
+
+    episodeContent.append(episodeTitle, episodeImage, episodeSummary);
+    ourSection.append(episodeContent);
+
+    addContentToEachEpisode(episodesList, episodeIndex);
+  }
+}
+
+function addContentToEachEpisode(list, index) {
+  let title = document.getElementsByClassName("episodeName")[index];
+  title.innerHTML = `${list[index].name} - <span> ${getSeasonAndEpisodeNumber(list, index)} </span>`;
```

Comment:
I'm curious, what did you choose to put the episode number in a `<span>` tag? I couldn't see any specific styling on the `<span>`

Reply:
@louisechow you are right, I followed last week challenge example (alarm clock) where I had the variable inside the a span tag, but as you said I do not need to do this.